request_a_govuk_domain/request/views.py

```diff
@@ -621,3 +621,7 @@ def service_failure_view(request):
     )
```

Comment:
I know this is not one of the changed lines, is this a GDPR violation to dump the data that may include user information?

Reply:
indeed. Removing